test/unit/io/sequence/sequence_file_out_test.cpp

```diff
@@ -330,6 +330,27 @@ TEST(row, emplace_back)
     });
 }
 
+TEST(row, different_fields)
```

Comment:
maybe a little more verbose: `different_fields_in_record_and_file` and two comment lines describing the situation?

Reply:
Okay, I added a comment and changed the name.